fw/t/unit/test_http2_parser_hpack.c

```diff
@@ -0,0 +1,269 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2014 NatSys Lab. (info@natsys-lab.com).
+ * Copyright (C) 2015-2022 Tempesta Technologies, Inc.
```

Comment:
This is a new file, so please leave only `Copyright (C) 2022 Tempesta Technologies, Inc.`